lib/manager/composer/artifacts.spec.ts

```diff
@@ -1,7 +1,8 @@
 import { exec as _exec } from 'child_process';
+import _fs from 'fs-extra';
 import { join } from 'upath';
 import { envMock, mockExecAll } from '../../../test/execUtil';
-import { fs, git, mocked } from '../../../test/util';
+import { git, mocked } from '../../../test/util';
```

Comment:
Why this change? why do you moch `fs-extra` instead of our `fs` wrapper?

Reply:
@viceice I followed the example of `gomod` testing which appeared to be more recent. See https://github.com/renovatebot/renovate/blob/master/lib/manager/gomod/artifacts.spec.ts#L14

Reply:
I think that's wrong, can you please revert to mock `util/fs` again. We need to change gomod later too.